package.json

```diff
@@ -4,8 +4,9 @@
   "description": "JS Assignment Repository",
   "main": "colorCoder.js",
   "scripts": {
-    "start": "node app/colorCoder.js"
+    "start": "node app/src/colorCoder.js"
```

Comment:
The intention is to run the test, so we can change this to `node app/test/ColorPairTest.js`